tequilapi/endpoints/transactor.go

```diff
@@ -658,64 +656,6 @@ func (te *transactorEndpoint) ChainSummary(c *gin.Context) {
 	})
 }
 
-// EligibilityResponse represents the eligibility response
-// swagger:model EligibilityResponse
-type EligibilityResponse struct {
-	Eligible bool `json:"eligible"`
-}
-
-// swagger:operation GET /transactor/identities/{id}/eligibility Eligibility
-// ---
-// summary: Checks if given id is eligible for free registration
-// parameters:
-// - name: id
-//   in: path
-//   description: Identity address to register
-//   type: string
-//   required: true
-// responses:
-//   200:
-//     description: Eligibility response
-//     schema:
-//       "$ref": "#/definitions/EligibilityResponse"
-//   500:
-//     description: Internal server error
-//     schema:
-//       "$ref": "#/definitions/APIError"
-func (te *transactorEndpoint) FreeRegistrationEligibility(c *gin.Context) {
```

Comment:
I dont really see a big reason to remove this. 🤷🏻  Could prove to be useful again in the future as well. Would be interesting to hear what other people think. @soffokl @Snawoot @n10ty 

This is also a breaking change which not only breaks webUI but probably also apps.

Reply:
also @mdomasevicius will be interested in this discussion I think

Reply:
Also, keep in mind that one of the endpoints is for providers, and therefore only place it can break is node UI and it's not used anymore there.

Reply:
@tomasmik you can emulate free registration with `/registration-payment` endpoint